### ISSUE TITLE
Remove expand/collapse icon when node has no more children

### ISSUE THREAD

**Issue by atolucky6** [vitmalina/w2ui#2307]
**Describe the problem your feature will solve.**
Should remove expand/collapse icon of parent node when remove all children node in sidebar control

**Describe the solution you'd like**
update collapsible of node to false when nodes.length === 0 after execute remove function
![Untitled](https://user-images.githubusercontent.com/61114829/192842913-eb7503a0-f7a9-4d83-8d11-db6edb0a0b33.png)

**Describe alternatives you've considered**

**Additional context**

**Comment by vitmalina** [comment]
Please provide a JS fiddle with this problem. I cannot see it here http://localhost:3500/demos/#/sidebar/1

**Comment by atolucky6** [comment]
Yes you can see it in w2ui demo by this link [https://github.com/vitmalina/w2ui/issues/2307](url)
To procedure the issue:
1. Click to Insert Items
2. Click to Remove Insert Items 

After remove insert items the node 'Level 1.2' has no more children but the expand/collapse icon still exists. It should be hidden after remove all children

**Comment by blitz-research** [comment]
The demo is working fine here, ie: the 'up arrow' is removed after removing inserted items.
